htdocs/product/list.php

```diff
@@ -1351,8 +1353,10 @@
 
 
 $product_static = new Product($db);
-$static_ws = new Workstation($db);
 $product_fourn = new ProductFournisseur($db);
+if (isModEnabled('workstation')) {
+	$static_ws = new Workstation($db);
```

Comment:
Same here, there is no table access.

Reply:
@eldy with this fix the error disappear! Try

Reply:
@eldy the "construct" function of workstation class call the "getNextNumRef" of this same class !! and access table is call !